docs/how-to-guides/streams/manage-streams-lab.md

```diff
@@ -0,0 +1,148 @@
+---
+id: manage-streams-lab
+title: How to manage data streams from Memgraph Lab
+sidebar_label: Manage data streams from Memgraph Lab
+---
+
+If you prefer to use GUI, you can connect to data streams by using a wizard in
+the **Stream** section of Memgraph Lab. If you prefere writing commands, you can
+[manage streams with queries](/how-to-guides/streams/manage-streams.md). 
+
+If you need a Kafka stream to play around with, we've provided some at [Awesome
+Data Stream](https://awesomedata.stream/)! 
+
+[![Related - Reference Guide](https://img.shields.io/static/v1?label=Related&message=Reference%20Guide&color=yellow&style=for-the-badge)](/reference-guide/streams/overview.md) [![Related -
+Tutorial](https://img.shields.io/static/v1?label=Related&message=Tutorial&color=008a00&style=for-the-badge)](/tutorials/graph-stream-processing-with-kafka.md)
+
+## How to add a stream?
+
+To add a stream in Memgraph Lab: 
+
+1. Switch to **Streams** and **Add New Stream**.
+2. Choose stream type, enter a stream name, server address, and topics you want to subscribe to.
+3. Go to the **Next Step**.
+4. Click on **Edit** (pencil icon) to modify the *Consumer Group*, *Batch

```

Comment:
I feel a bit insulted with this pencil icon comment.🥲 It should be clear from UI prespective, if it is not we are using wrong icons in LAB 😂

Reply:
Ah, the constant struggle to be sure what lvl are our users :D :D ok, ok... I'll remove it